core/request_test.go

```diff
@@ -1,11 +1,14 @@
 package core_test
 
 import (
+	"context"
 	"encoding/base64"
 	"io/ioutil"
 	"math/rand"
 	"os"
 
+	"github.com/aws/aws-lambda-go/lambdacontext"
+
```

Comment:
there doesn't need to be a blank-line here. It's from the same repo/company as the imports below it, and isn't different enough from those later imports to belong in its own stanza.

Reply:
Not intentional, corrected.